apps/src/templates/progress/progressStyles.js

```diff
@@ -65,28 +106,30 @@ const statusStyle = {
  * Given a level object, figure out styling related to its color, border color,
  * and background color
  */
-export const levelProgressStyle = (level, disabled) => {
+export const levelProgressStyle = (levelStatus, levelKind, disabled) => {
   let style = {
-    borderWidth: 2,
+    borderWidth: BUBBLE_BORDER_WIDTH,
+    borderColor: color.lighter_gray,
+    borderStyle: 'solid',
     color: color.charcoal,
     backgroundColor: color.level_not_tried
   };
 
-  if (disabled) {
-    style = {
-      ...style,
-      ...(!disabled && hoverStyle)
-    };
-  } else {
-    if (level.status !== LevelStatus.not_tried) {
-      style.borderColor = color.level_perfect;
-    }
+  if ((disabled && levelStatus !== LevelStatus.submitted) || !levelStatus) {
```

Comment:
Are there cases where existing bubbles will change due to to this change in logic?  (I'm having trouble convincing myself either way.)

Reply:
yes, good question. i added this check so that submitted assessments still show their submitted status even when disabled, since they are usually disabled after students have submitted them. will add a comment.